Desktop/Test_Plan_VFS.md

```diff
@@ -70,7 +70,7 @@ Test environment:
 
 
 * [ ] Test with a long path (make sure LongPathsEnabled in Windows registry is off ** )
-  * [ ] Create a file with a long path (> 260)
+  * [ ] Create a file inside some folders with a path length > 260 chars (> MAX_PATH length)
```

Comment:
Maybe explicitly mention that we don't mean long file names.

This test is not vfs specific, the issue occurred recently with vfs but can affect any part of the client.